drivers/vulkan/rendering_device_driver_vulkan.cpp

```diff
@@ -5668,12 +5726,8 @@ uint64_t RenderingDeviceDriverVulkan::api_trait_get(ApiTrait p_trait) {
 
 bool RenderingDeviceDriverVulkan::has_feature(Features p_feature) {
 	switch (p_feature) {
-		case SUPPORTS_MULTIVIEW:
```

Comment:
Same here, this looks like an accidental delete

Reply:
Never mind, noticed the code had already moved to `RenderingDevice::has_feature`, so this seems to be some cleanup